### DISCUSSION TITLE
Any one has been updated to monterey release version?

### DISCUSSION THREAD

**Discussion by icespring** [osy86/HaC-Mini#946] [General]
Any one has been updated to monterey release version?

I want to know if update to monterey works well

**Comment by MokOopsing** [comment]
I update, now loop in restart

**Reply by osy** [reply]
You are on v3.1.2?

**Reply by MokOopsing** [reply]
> You are on v3.1.2?

yes, update from big sur with hac-mini 3.1.2, I filed an issue #942, but no one reply yet

**Reply by khanhdt123** [reply]
I have the same problem after updating to Monterey Beta 10. After checking, I uncheck on Intel Bluetooth, I can work with Monterey but lost Bluetooth. (The Stock intel kext still work on Big Sur before the update)

<img width="633" alt="Screen Shot 2021-10-26 at 6 09 21 PM" src="https://user-images.githubusercontent.com/55571752/138866191-c8c99097-2198-4c86-95db-4b3a5a247d31.png">

**Reply by goodbest** [reply]
> I have the same problem after updating to Monterey Beta 10. After checking, I uncheck on Intel Bluetooth, I can work with Monterey but lost Bluetooth. (The Stock intel kext still work on Big Sur before the update)
> 
> <img alt="Screen Shot 2021-10-26 at 6 09 21 PM" width="633" src="https://user-images.githubusercontent.com/55571752/138866191-c8c99097-2198-4c86-95db-4b3a5a247d31.png">

The bluetooth issue (with intel / brcm card, not apple card) may be related to this.
https://github.com/acidanthera/bugtracker/issues/1821

Try to use this kext after v3.1.2 to see if the issue is gone.
https://github.com/dortania/build-repo/releases/download/BrcmPatchRAM-44c3025/BrcmPatchRAM-2.6.1-RELEASE.zip

**Reply by MokOopsing** [reply]
I remove the Intel bluetooth part from the config.plist, and put it back to the efi/oc, still loop in restart

**Comment by icespring** [comment]
Apple has released Monterey 12.0.1 today.  Has any one works well on 12.0.1 ?

**Comment by keatliang2005** [comment]
Update from Big Sur 11.6 to 12.0.1 all goes well

using Fenvi AC12200 supported OOB card
